moviebuff.go

```diff
@@ -290,3 +292,41 @@ func (m *moviebuff) GetCertifications(country string) ([]Certification, error) {
 
 	return certifications.Data, nil
 }
+
+func (m *moviebuff) GetHolidayCalendar(countryID string) (*Calendar, error) {
+	r, err := prepareRequest(m.HostURL, m.StaticToken, "/holidays/"+countryID)
+	if err != nil {
+		return nil, err
+	}
+
+	res, err := new(http.Client).Do(r)
+	if err != nil {
+		return nil, err
+	}
+	defer res.Body.Close()
+
+	if res.StatusCode != http.StatusOK {
```

Comment:
This block can be simplified by using a switch case

Reply:
@svetha-cvl  Changed to switch case. Please review. @ilayaraja89